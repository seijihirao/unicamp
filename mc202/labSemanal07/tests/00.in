A A
ab ba
Yxz xYz
fxy yxf
abc bac
weq qew
abg bga
acd adc
abt bta
poeq eopq
qwte wtqe
cdef fedc
ACBD ABDC
ABCDEF CBAEDF
