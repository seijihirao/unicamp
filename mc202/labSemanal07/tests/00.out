A A
ba ab
xzY Yxz
yxf fxy
bca abc
qew weq
gba abg
dca acd
tba abt
eoqp poqe
tweq qwet
fedc cdef
DBCA ACBD
CBEFDA ABDCEF
